flow-launcher.tera

```diff
@@ -0,0 +1,201 @@
+        x:Key="ItemTitleSelectedStyle"
+        BasedOn="{StaticResource BaseItemTitleSelectedStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Cursor" Value="Arrow" />
+        <Setter Property="Foreground" Value="#{{ text.hex }}" />
+    </Style>
+    <Style
+        x:Key="ItemSubTitleSelectedStyle"
+        BasedOn="{StaticResource BaseItemSubTitleSelectedStyle}"
+        TargetType="{x:Type TextBlock}">
+        <Setter Property="Cursor" Value="Arrow" />
+        <Setter Property="Foreground" Value="#{{ blue.hex }}" />
+    </Style>
+    <SolidColorBrush x:Key="ItemSelectedBackgroundColor">#{{ if(cond=flavor.dark, t=base.hex, f=surface0.hex) }}</SolidColorBrush>
+    <CornerRadius x:Key="ItemRadius">8</CornerRadius>
+    <Thickness x:Key="ItemMargin">10 0 10 0</Thickness>
+    <Style
+        x:Key="ItemImageSelectedStyle"
+        BasedOn="{StaticResource BaseItemImageSelectedStyle}"
+        TargetType="{x:Type Image}">
+        <Setter Property="Cursor" Value="Arrow" />
+    </Style>
+    <Style x:Key="HighlightStyle">
+        <Setter Property="Inline.Foreground" Value="#{{ mauve.hex }}" />
+        <Setter Property="Inline.FontWeight" Value="Bold" />
+    </Style>
+    <Style x:Key="ItemHotkeyStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="FontSize" Value="13" />
+        <Setter Property="Foreground" Value="#{{ overlay0.hex }}" />
+    </Style>
+    <Style x:Key="ItemHotkeySelectedStyle" TargetType="{x:Type TextBlock}">
+        <Setter Property="FontSize" Value="13" />
+        <Setter Property="Foreground" Value="#{{ blue.hex }}" />
+    </Style>
+    <!--  button style in the middle of the scrollbar  -->
+    <Style
+        x:Key="ThumbStyle"
+        BasedOn="{StaticResource BaseThumbStyle}"
+        TargetType="{x:Type Thumb}">
+        <Setter Property="SnapsToDevicePixels" Value="True" />
+        <Setter Property="OverridesDefaultStyle" Value="true" />
+        <Setter Property="IsTabStop" Value="false" />
+        <Setter Property="Width" Value="2" />
+        <Setter Property="Focusable" Value="false" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="{x:Type Thumb}">
+                    <Border
+                        Background="#{{ surface0.hex }}"
+                        BorderBrush="Transparent"
+                        BorderThickness="0"
+                        CornerRadius="2"
+                        DockPanel.Dock="Right" />
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+    <Style
+        x:Key="ScrollBarStyle"
+        BasedOn="{StaticResource BaseScrollBarStyle}"
+        TargetType="{x:Type ScrollBar}" />
+    <Style
+        x:Key="SeparatorStyle"
+        BasedOn="{StaticResource BaseSeparatorStyle}"
+        TargetType="{x:Type Rectangle}">
+        <Setter Property="Fill" Value="#{{ if(cond=flavor.dark, t=base.hex, f=surface0.hex) }}" />
```

Comment:
Maybe we can just make this `surface0` and see how it looks on all flavours? That's what we do in userstyles for separators right?